test/extended/router/h2spec.go

```diff
@@ -340,3 +345,11 @@ func lookupEnv(key, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func clusterWideProxyIsConfigured(oc *exutil.CLI) (bool, error) {
```

Comment:
Can we just use `IsClusterProxyEnabled` from `test/extended/util/proxy.go`?

Reply:
Done in https://github.com/openshift/origin/commit/fde50de9606c8aa17249d606e39dda78032a9c75. Thanks.

Reply:
In https://github.com/openshift/origin/pull/26086/commits/9fd32dceb0e243bb2ca4739b4bcab2a0f83520ee.